app/controllers/project/conversations.js

```diff
@@ -1,4 +1,11 @@
 import Controller from '@ember/controller';
+import { set } from '@ember/object';
 
 export default Controller.extend({
+  actions: {
+    close(conversation) {
+      set(conversation, 'status', 'closed');
+      conversation.save();
```

Comment:
I might consider adding a `return` here in the event we ever `pipe` this, so the promise returns/rejects.